src/ApiPort/CommandLine/AnalyzeOptions.cs

```diff
@@ -117,6 +119,17 @@ private void UpdateRequestFlags(Options options)
                     RequestFlags |= AnalyzeRequestFlags.ShowBreakingChanges;
                 }
 
+                if (options.ShowRetargettingIssues)
+                {
+                    RequestFlags |= AnalyzeRequestFlags.ShowRetargettingIssues;
+
+                    //ShowRetargettingIssues doesn't mean much without ShowBreakingChanges turned on
+                    if (!RequestFlags.HasFlag(AnalyzeRequestFlags.ShowBreakingChanges))
```

Comment:
This will work, but for what it's worth |= the same thing multiple times is a no-op, so you could just add the ShowBreakingChanges flag unconditionally here.